_opensearch/supported-field-types/xy-shape.md

```diff
@@ -0,0 +1,379 @@
+  "location" : "MULTIPOINT (0.5 4.5, 2.5 6.0)"
+}
+```
+
+## Multiline string
+
+A multiline string is an array of line strings.
+
+Index a line string in GeoJSON format:
+
+```json
+PUT testindex/_doc/2
+{
+  "location" : {
+    "type" : "multilinestring",
+    "coordinates" : [
+      [[0.5, 4.5], [2.5, 6.0]],
+      [[1.5, 2.0], [3.5, 3.5]]
+      ]
+  }
+}
+```
+
+Index a line string in WKT format:
+
+```json
+PUT testindex/_doc/2
+{
+  "location" : "MULTILINESTRING ((0.5 4.5, 2.5 6.0), (1.5 2.0, 3.5 3.5))"
+}
+```
+
+## Multi polygon
+
+A multi polygon is an array of polygons. In this example, the first polygon contains a hole, and the second does not. 
+
+Index a multi polygon in GeoJSON format:
+
+```json
+PUT testindex/_doc/4
+{
+  "location" : {
+    "type" : "multipolygon",
+    "coordinates" : [
+    [
+      [[0.5, 4.5], 
+      [2.5, 6.0], 
+      [1.5, 2.0], 
+      [0.5, 4.5]],
+      
+      [[1.0, 4.5], 
+      [1.5, 4.5], 
+      [1.5, 4.0], 
+      [1.0, 4.5]]
+    ],
+    [
+      [[2.0, 0.0], 
+      [1.0, 2.0], 
+      [3.0, 1.0], 
+      [2.0, 0.0]]
+      ]
+    ]
+  }
+}
+```
+
+Index a multi polygon in WKT format:
+
+```json
+PUT testindex/_doc/4
+{
+  "location" : "MULTIPOLYGON (((0.5 4.5, 2.5 6.0, 1.5 2.0, 0.5 4.5), (1.0 4.5, 1.5 4.5, 1.5 4.0, 1.0 4.5)), ((2.0 0.0, 1.0 2.0, 3.0 1.0, 2.0 0.0)))"
+}
+```
+
+## Geometry collection
+
+A geometry collection is a collection of xy shapes that may be of different types.
+
+Index a geometry collection in GeoJSON format:
+
+```json
+PUT testindex/_doc/7
+{
+  "location" : {
+    "type": "geometrycollection",
+    "geometries": [
+      {
+        "type": "point",
+        "coordinates": [0.5, 4.5]
+      },
+      {
+        "type": "linestring",
+        "coordinates": [[2.5, 6.0], [1.5, 2.0]]
+      }
+    ]
+  }
+}
+```
+
+Index a geometry collection in WKT format:
+
+```json
+PUT testindex/_doc/7
+{
+  "location" : "GEOMETRYCOLLECTION (POINT (0.5 4.5), LINESTRING(2.5 6.0, 1.5 2.0))"
+}
+```
+
+## Envelope
+
+An envelope is a bounding rectangle specified by top-left and bottom-right vertices. The GeoJSON format is `[[minX, maxY], [maxX, minY]]`.
+
+Index an envelope in GeoJSON format:
+
+```json
+PUT testindex/_doc/2
+{
+  "location" : {
+    "type" : "envelope",
+    "coordinates" : [[3.0, 2.0], [6.0, 0.0]]
+  }
+}
+```
+
+In WKT format, use `BBOX (minX, maxY, maxX, minY)`.
+
+Index an envelope in WKT BBOX format:
+
+```json
+PUT testindex/_doc/8
+{
+  "location" : "BBOX (3.0, 2.0, 6.0, 0.0)"
+}
+```
+
+## Parameters
+
+The following table lists the parameters accepted by xy shape field types. All parameters are optional.
+
+Parameter | Description 
+:--- | :--- 
+`coerce` | A Boolean value that specifies whether to automatically close unclosed linear rings. Default is `false`.
+`ignore_malformed` | A Boolean value that specifies to ignore malformed GeoJSON or WKT xy shapes and not to throw an exception. Default is `false` (throw an exception when xy shapes are malformed).
```

Comment:
should it be **throws** ?

Reply:
I think it should be "throw" because it's saying the default is to throw an exception. But let's wait for @natebower's review.

Reply:
They're both technically correct because these sentences are fragments, so they're dependent on omitted text, which could be whatever we want it to be. I vote for "throw" in order to maintain consistency with the preceding sentence.